app/src/main/java/com/example/habitsmasher/ImageDatabaseHelper.java

```diff
@@ -19,7 +19,7 @@
     private StorageReference _storageReference = _storage.getReference();
 
     // Max download megabyte size
-    private final long ONE_MEGABYTE = 1024 * 1024;
+    private final long ONE_MEGABYTE = 1024 * 1024 * 5;
```

Comment:
Is this no longer `ONE_MEGABYTE`? Is it now `FIVE_MEGABYTES`?

Reply:
Good catch